docs/README.md

```diff
@@ -10,29 +10,29 @@ Welcome to the documentation of the **Cosmos Hub application: `gaia`**.
 
 ## What is Gaia?
 
-- [Intro to the `gaia` software](./getting-started/what-is-gaia.md)
-- [Interacting with the `gaiad` binary](./hub-tutorials/gaiad.md)
+- [Intro to the `gaia` software](/getting-started/what-is-gaia.md)
```

Comment:
These links no longer work. See https://github.com/cosmos/gaia/blob/mmulji-ic/fix-broken-links-in-docs-2/docs/README.md